WORKSPACE

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+workspace(name = "org_abseil_py")
```

Comment:
Thanks, can you change this to io_abseil_py since the website is abseil.io.

Reply:
Sure. done.